python/tvm/contrib/target/onnx.py

```diff
@@ -617,6 +617,20 @@ def convert_attributes(cls, attrs):
         return {"value": 1}
 
 
+class LRN(OpConverter):
+    """Operator converter for LRN."""
+
+    @classmethod
+    def convert_attributes(cls, attrs):
+        return {
+            "alpha": attrs.alpha,
+            "beta": attrs.beta,
+            "bias": attrs.bias,
+            "size": attrs.size
+            # axis?
```

Comment:
Remove comment.

It looks like the ONNX LRN op doesn't have an axis parameter, as it always applies to axis 1.
Should be add an assert somewhere to check that the axis is also 1 in the Relay operator that is being converted?

Reply:
Have added the axis check.